baseapp/logging.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package baseapp
+
+import (
+	"io"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+)
+
+// ConfigureDefaultLogger returns a zerolog logger based on the conventions in a LoggingConfig
+func ConfigureDefaultLogger(c LoggingConfig) (zerolog.Logger, error) {
```

Comment:
`NewLogger`? Using "configure" implies there is an existing logger that is being changed. I'm also not sure "default" is needed: most other uses of "default" in this library are for functions that also have a version taking more arguments or for functions that are called as part of a larger "default" function.